packages/web-components/src/accordion/accordion-item/accordion-item.styles.ts

```diff
@@ -0,0 +1,236 @@
+import { css } from '@microsoft/fast-element';
+import {
+  borderRadiusMedium,
+  borderRadiusSmall,
+  colorNeutralForeground1,
+  colorNeutralForegroundDisabled,
+  colorStrokeFocus1,
+  colorStrokeFocus2,
+  colorTransparentBackground,
+  fontFamilyBase,
+  fontSizeBase200,
+  fontSizeBase300,
+  fontSizeBase400,
+  fontSizeBase500,
+  fontWeightRegular,
+  lineHeightBase200,
+  lineHeightBase300,
+  lineHeightBase400,
+  lineHeightBase500,
+  spacingHorizontalM,
+  spacingHorizontalMNudge,
+  spacingHorizontalS,
+} from '../../theme/design-tokens.js';
+
+export const styles = css`
+  /* --- Default styles --- */
+
+  :host {
+    max-width: fit-content;
+  }
+
+  :host .heading {
+    height: 44px;
+    display: grid;
+    position: relative;
+    vertical-align: middle;
+    padding-left: ${spacingHorizontalMNudge};
+    padding-right: ${spacingHorizontalM};
+    border-radius: ${borderRadiusMedium};
+    font-family: ${fontFamilyBase};
+    font-size: ${fontSizeBase300};
+    font-weight: ${fontWeightRegular};
+    line-height: ${lineHeightBase300};
+    grid-template-columns: 28px 28px auto 28px;
+  }
+
+  :host .heading .heading-content {
+    height: 100%;
+    display: flex;
+    align-items: center;
+  }
```

Comment:
Same question here as above, but specific to including `.heading` - if there's a class for `.heading-content` and we don't need the specificity of the preceding selector we should remove it. The first paragraph or so of [this blog](https://blogs.windows.com/msedgedev/2023/01/17/the-truth-about-css-selector-performance/) details how the browser engine handles this scenario. TLDR, it's going to evaluate from right to left, so it's going to look for `.heading-content` and then before applying anything look up the tree to find `.heading` before reconciling (and so on for a selector). We're likely doubling up on the cost to apply these styles.